schemas/2018-04-01/Microsoft.Databricks.json

```diff
@@ -429,6 +429,39 @@
           ],
           "description": "The object that contains details of encryption used on the workspace."
         },
+        "loadBalancerBackendPoolName": {
```

Comment:
Would you mind reverting all the auto-generated files in this PR? They should be regenerated automatically once it's been merged.

Reply:
Sure, makes sense. Reverted.